addons/pos_loyalty/tests/test_frontend.py

```diff
@@ -2148,3 +2148,32 @@ def test_loyalty_reward_product_tag(self):
             "PosLoyaltyRewardProductTag",
             login="accountman"
         )
+
+    def test_loyalty_on_order_with_fixed_tax(self):
+
+        self.env['loyalty.program'].search([('id', '!=', self.auto_promo_program_next.id)]).write({'active': False})
+        self.auto_promo_program_next.coupon_ids = [Command.create({
+            'code': '563412',
+            'points': 10
+        })]
+
+        fixed_tax = self.env['account.tax'].create({
+            'name': 'Fixed Tax',
+            'amount_type': 'fixed',
+            'amount': 50,
+        })
+        self.env["product.product"].create(
+            {
+                "name": "Product A",
+                "type": "product",
+                "list_price": 15,
+                "available_in_pos": True,
+                "taxes_id": [Command.link(fixed_tax.id)],
+            }
+        )
+
+        self.start_tour(
```

Comment:
I'm suprised the tour works without opening the PoS before the tour. You should probably open the PoS like this `self.main_pos_config.open_ui()`